content/how-to/shim.md

```diff
@@ -1,8 +1,113 @@
 ---
-title: How to Shim?
+title: How to Shim Third party libraries?
```

Comment:
Maybe do titlecase so we get "How to Shim Third Party Libraries?". Consistent that way.